wpimath/src/main/java/edu/wpi/first/math/kinematics/SwerveDriveKinematics.java

```diff
@@ -41,7 +44,9 @@
  */
 @SuppressWarnings("overrides")
 public class SwerveDriveKinematics
-    implements Kinematics<SwerveModuleState[], SwerveModulePosition[]> {
+    implements Kinematics<SwerveModuleState[], SwerveModulePosition[]>,
+        ProtobufSerializable,
+        StructSerializable {
```

Comment:
Marked as StructSerializable, but there's no struct implementation.

Reply:
Oops, I'll add that. Might take a while, though, since I'm busy for most of today.